packtools/sps/validation/article_citations.py

```diff
@@ -114,23 +111,22 @@ def validate_article_citation_article_title(self):
                 },...
             ]
         """
-        for citation in self.article_citations:
-            publication_type = citation.get('publication_type')
-            if publication_type == 'journal':
-                article_title = citation.get('article_title')
-                is_valid = article_title is not None
-                yield {
-                    'title': 'element citation validation',
-                    'element': 'element-citation',
-                    'sub-element': 'article-title',
-                    'validation_type': 'exist',
-                    'response': 'OK' if is_valid else 'ERROR',
-                    'expected_value': article_title if is_valid else 'a valid value to article-title',
-                    'got_value': article_title,
-                    'message': f'Got {article_title} expected {article_title if is_valid else "a valid value to article-title"}',
-                    'advice': None if is_valid else f"The article-title in reference (ref-id: {citation.get('ref_id')}) is missing "
-                                                    f"provide a valid value to article-title"
-                }
+        publication_type = self.citation.get('publication_type')
+        if publication_type == 'journal':
+            article_title = self.citation.get('article_title')
+            is_valid = article_title is not None
+            yield {
+                'title': 'element citation validation',
+                'element': 'element-citation',
+                'sub-element': 'article-title',
+                'validation_type': 'exist',
+                'response': 'OK' if is_valid else 'ERROR',
+                'expected_value': article_title if is_valid else 'a valid value to article-title',
+                'got_value': article_title,
+                'message': f'Got {article_title} expected {article_title if is_valid else "a valid value to article-title"}',
```

Comment:
for no lugar de to, porque article-title é um substantivo

Reply:
OK, @robertatakenaka